app/controllers/taxons_controller.rb

```diff
@@ -21,7 +21,7 @@ def redirect_to_www
   def taxon
     @taxon ||= begin
       taxon_base_path = params[:taxon_base_path]
-      Taxon.find('/alpha-taxonomy/' + taxon_base_path)
+      Taxon.find("/" + taxon_base_path)
```

Comment:
Ok, I will use the `request.path` then in order to avoid manipulating the param, otherwise we would still need to do this: https://github.com/alphagov/content-store/blob/master/app/controllers/application_controller.rb#L26-L28